torch/_inductor/codecache.py

```diff
@@ -1217,6 +1218,10 @@ def post_compile(
         """
         set_tracing_context_output_strides(example_inputs, compiled_graph)
 
+        # If we have not ended compile using TritonBundler.collect, lets
+        # end here.
```

Comment:
@aorenste Doesn't this mean Triton autotune bundled cache also has this same problem?